### PR TITLE
Swift 4 support

### PR DESCRIPTION
**Pull request by michaelloo**
# Overall

* Update project to Swift 4
* Enable Xcode suggested warnings
* Set deployment target to 9.0

### CODE REVIEW

File: Scenarios.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -363,13 +371,13 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```
Comment (sharplet) [inline Scenarios.xcodeproj/project.pbxproj]: Curious, were there any errors/warnings related to targeting 8.0?

**Review by sharplet — COMMENTED** [review]
This looks good to me! Just one question about deployment target.

Could you please update this PR to target `master` instead of `develop`? I don't think we need a `develop` branch on this repo.

File: Scenarios.xcodeproj/project.pbxproj (hunk 1 above)
Comment (michaelloo) [reply Scenarios.xcodeproj/project.pbxproj]: Nope, nothing on Xcode at least!

**Comment by michaelloo** [comment]
@sharplet pointed this PR to `master`. I'll also delete `develop`.

File: Scenarios.xcodeproj/project.pbxproj (hunk 1 above)
Comment (sharplet) [reply Scenarios.xcodeproj/project.pbxproj]: May as well keep the same deployment target then?

**Review by sharplet — APPROVED** [review]
🚢

**Pull request merged**